tests/index.js

```diff
@@ -19,25 +19,6 @@ testPdData.body = {};
 
 describe('index.js >', () => {
   describe(' createTTE tests', () => {
-    it('should return a single tte acknowledge_log_entry', () => {
-      const expected = {
-        id: 'aa',
-        start: '2019-09-04T09:16:41Z',
-        end: '2019-09-04T09:16:41Z',
-        team: 'TestTeam'
-      };
-      testPdData.body.log_entries = [
-        {
-          type: 'notify_log_entry',
-          created_at: '2019-09-04T09:16:41Z',
-        },
-        {
-          type: 'acknowledge_log_entry',
-          created_at: '2019-09-04T09:16:41Z',
-        }];
-      const tte = createTTE('aa', testTeamData, testPdData);
-      expect(tte).to.deep.equal(expected);
-    });
     it('should return a single tte resolve_log_entry', () => {
       const expected = {
         id: 'ab',
```

Comment:
Can you change the start time here with a time before the end date, makes the test more readable?